### PR TITLE
Add new Database Migration Service (DMS) resources

### PR DESCRIPTION
**Pull request by ericklow**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

***

Add new Database Migration Service (DMS) resources

***

Release note for [CHANGELOG](https://github.com/traveloka/terraform-aws-resource-naming/blob/master/CHANGELOG.md):

```release-note
FEATURES:

* **New Source:** `dms_endpoint`
* **New Source:** `dms_replication_instance`
* **New Source:** `dms_replication_task`
```

***

Output from `terraform plan` command from changes you propose.

```
$ terraform plan

```

### CODE REVIEW

File: resources.tf [hunk 1]
```diff
@@ -12,6 +12,9 @@ locals {
     "codepipeline"                        = "100" # https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_PipelineDeclaration.html#CodePipeline-Type-PipelineDeclaration-name
     "db_instance"                         = "63"  # https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html
     "db_parameter_group"                  = "255" # https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBParameterGroup.html
+    "dms_endpoint"                        = "255" # https://docs.aws.amazon.com/dms/latest/APIReference/API_CreateEndpoint.html
+    "dms_replication_instance"            = "255" # https://docs.aws.amazon.com/dms/latest/APIReference/API_CreateReplicationTask.html
+    "dms_replication_task"                = "255" # https://docs.aws.amazon.com/dms/latest/APIReference/API_CreateReplicationInstance.html
```
Comment (nin9swells) [inline resources.tf]: - Documentation is swapped between dms_replication_instance and dms_replication_task. Please fix
- dms_replication_instance is only 63 character long

File: resources.tf [hunk 2]
```diff
@@ -12,6 +12,9 @@ locals {
     "codepipeline"                        = "100" # https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_PipelineDeclaration.html#CodePipeline-Type-PipelineDeclaration-name
     "db_instance"                         = "63"  # https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html
     "db_parameter_group"                  = "255" # https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBParameterGroup.html
+    "dms_endpoint"                        = "255" # https://docs.aws.amazon.com/dms/latest/APIReference/API_CreateEndpoint.html
```
Comment (nin9swells) [inline resources.tf:15]: Since the documentation doesn't tell us limit of dms_endpoint, please ask AWS support about this before we decide to use 255.

**Review by nin9swells — CHANGES_REQUESTED** [review]

File: resources.tf (hunk 1 above)
Comment (ericklow) [reply resources.tf]: Updated 
- https://github.com/traveloka/terraform-aws-resource-naming/pull/61/commits/26c7ad413606f46ed2c479509c14ef35170ee1de
- https://github.com/traveloka/terraform-aws-resource-naming/pull/61/commits/86230933065432b43f63ae3f65706f0944e699f9

File: resources.tf (hunk 2 above)
Comment (ericklow) [reply resources.tf:15]: Confirmed that it is 255
image. Thanks.

**Review by nin9swells — APPROVED** [review]

**Review by nin9swells — APPROVED** [review]

**Review by nin9swells — APPROVED** [review]

**Pull request merged**